### DISCUSSION TITLE
Retrieving deleted table and project

### DISCUSSION THREAD

**Discussion by abhay187** [supabase/supabase#9808] [Questions]
Hello,
I was wondering if there is a way to retrieve deleted table or project in case I accidentally or some malicious element deleted it. This feature seems a must for a production project.  Firebase allows 30 days to retrieve deleted project

Thanks

**Comment by iAmWillShepherd** [comment]
I'm also curious about this, as I just blew away my `Auth.users` table.

**Reply by GaryAustin1** [reply]
If you are not on a pro plan with backup, then your only hope is to contact support and see if they can help.
